service-sdk/12.0.0/add-persistence-to-service/example-service/pom.xml

```diff
@@ -20,7 +20,7 @@
     <name>Backbase :: Digital Banking Services :: example-service</name>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```

Comment:
Do we actually need to specify java version?

Reply:
No, it's part of the SSDK, but not sure if we had it here on purpose to make more explicit so I just updated it